scripts/install/installnode.py

```diff
@@ -0,0 +1,463 @@
+from JumpScale import j
+import requests
+import os
+
+class Node():
+    """ Install basic packages on a node """
+    def __init__(self, config):
+        self.config = config
+
+    def prepare_config(self):
+        self.nginx = {
+            'servicename': 'nginx',
+            'instance': 'system',
+            'daemonpath': './nginx ',
+            'args': '-c /opt/nginx/cfg/nginx.conf',
+            'pwd': "/opt/nginx",
+            'instance': 'main'
+        }
+
+        self.redis = {
+            'config': {
+                "name": "main",
+                "ip": "0.0.0.0",
+                "port": "9999",
+                "maxram": "100",
+                "disk": "0",
+                "passwd": "",
+                "unixsocket": "0"
+            },
+            'servicename': 'redis',
+            'instance': 'system',
+            'daemonpath': '/opt/jumpscale7/apps/redis/redis-server',
```

Comment:
we decided to reuse the redis package from ubuntu but because OVS also uses this redis we will leave the default installed systemd service as is and register a 2nd one pointing to OUR redis configuration (max memory should be 200MiB incase of storage role)